azure-pipelines.sds.master-release.yml

```diff
@@ -21,7 +21,7 @@ parameters:
     buildPushImage: true
     pushHelmChart: true
   - env: 'demo'
-    requireApproval: true
+    requireApproval: false
```

Comment:
looks like we might need to change this in admin web too, as it's currently set to true there

at least I take it that is the intention - require approval for all apps in test, but not demo?

Reply:
the require approval for demo doesn't actually do anything atm the moment because it's configured to use the same image as dev, so demo updates behind the scenes anyway despite the approval pending. I changed it though it does nothing for no reason other than visually being correct